capstone/client/src/components/ListPage.js

```diff
@@ -194,6 +187,16 @@ class ListPage extends Component {
     });   
   }
 
+  getItemTypes = () => {
+    axios.get(
```

Comment:
Add error handling if this request returns an error.

Reply:
Done!